pkg/services/ngalert/api/api_provisioning.go

```diff
@@ -620,3 +633,7 @@ func exportHcl(download bool, body definitions.AlertingFileExport) response.Resp
 	}
 	return resp.SetHeader("Content-Type", "text/hcl")
 }
+
+func saveTrans(ctx context.Context) error {
+	return nil
+}
```

Comment:
this does not seem to be used anywhere.

Reply:
It isn't, not sure where it came from either. Removing.